### PR TITLE
Adding Instalution

### PR DESCRIPTION
**Pull request by groovenectar**
The list of TODOs lives in the README file here: https://github.com/instalution/frontend/blob/master/README-en.md#user-content-short-list-of-needs--todos

Not opposed to other patterns

### CODE REVIEW

**Comment by groovenectar** [comment]
Is that first test checking for a website link in the Github profile? Is there a way to re-run that?

File: _data/projects/Instalution.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: Instalution aims to be a revolutionary solution to Instagram.
+site: https://github.com/instalution/frontend
+tags:
+  - artist
+  - media
+  - portfolio
+  - frontend
+  - developer-network
+upforgrabs:
+  name: instalution
```
Comment (ritwik12) [inline _data/projects/Instalution.yml]: @groovenectar This should be the name of a LABEL at your git repo.

File: _data/projects/Instalution.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  link: https://github.com/instalution/frontend/blob/master/README-en.md#short-list-of-needs--todos
```
Comment (ritwik12) [inline _data/projects/Instalution.yml]: @groovenectar This should be the link to that LABEL. Eg: https://github.com/instalution/frontend/labels/good%20first%20issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Instalution.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+desc: Instalution aims to be a revolutionary solution to Instagram.
+site: https://github.com/instalution/frontend
+tags:
+  - artist
+  - media
+  - portfolio
+  - frontend
+  - developer-network
+upforgrabs:
+  name: up-for-grabs
```
Comment (groovenectar) [inline _data/projects/Instalution.yml:11]: @[ritwik12](https://github.com/ritwik12) Is this the correct format?

**Review by ritwik12 — APPROVED** [review]

**Pull request merged**